DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchBasics/metadata.json

```diff
@@ -0,0 +1,128 @@
+{
+    "Slug": "fts-query-search-basics",
+    "SourceFileName": "FTSQuerySearchBasics.java",
+    "Title": "FTS Query - Basics",
+    "DescriptionLines": [
+        "**Full Text Search query** allows querying for documents which contain the specified term(s) within the searched document field.",
+        "",
+        "Upon query, an auto-index is created on the searched document fields."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Operation in queries - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#operations-in-queries"
+        },
+        {
+            "Title": "Full text searching - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#full-text-searching"
+        }
+        /* TODO
+        ,
+        {
+            "Title": "How to search",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+        },
+        {
+            "Title": "Querying searching",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching"
+        }
+        */
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Employees' collection using the Session `query()`."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type": "demo",
+                    "Url": "/java/queries/query-overview"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `search()` to search for the specified terms in the specified document field.",
+                "",
+                "* The most basic usage is to search for a single term, i.e.:<br>",
+                "  .search('Notes', 'searchTerm')",
+                "",
+                "* In this example, we search for term1 OR term2 in the document field _Notes_.",
```

Comment:
need to make it with an underscore - just like in the C# demo

`term1 OR term2` => `_term1 OR term2_`